src/Types/Vector.h

```diff
@@ -45,7 +45,7 @@ namespace ippl {
         
         Vector(const std::array<T, Dim>& a);
 
-	Vector(const std::array<std::vector<T>, Dim>& a);
+	    Vector(const std::array<std::vector<T>, Dim>& a);
```

Comment:
What happened here? Recommend rerunning `clang-format`.

Reply:
ah I see now. `clang-format` actually did that for me; however there's a tab in this line (not from me). I'll fix it with spaces.